lib/tarantool_server.py

```diff
@@ -449,7 +449,7 @@ def seek_wait(self, msg, proc=None, name=None):
         while True:
             if os.path.exists(self.path):
                 break
-            time.sleep(0.001)
+            gevent.sleep(0.001)
```

Comment:
Another point that disquiets me: whether a Popen handle gives a correct result, when we spin around `<Popen_handle>.pool()` and `gevent.sleep()`? I saw some strange effects and described them in #252. I would make an attempt to reveal a root of the problem and understand, whether a problem of this kind can appear here too. At least try and if'll not become clear, leave a note in #252 to revisit the server startup waiting code too when we'll investigate the issue.

Reply:
As a comment about time/gevent sleep, I spent some time clarify it, but haven't received any good results. I suggest shifting it for the next time.

Reply:
So, please, leave a note in the issue: we should revisit those time.sleep() -> gevent.sleep() changes in context of the described subprocess module problem.

Reply:
I [did](https://github.com/tarantool/test-run/issues/252#issuecomment-855380882) it.